packages/lesswrong/components/common/HomeLatestPosts.tsx

```diff
@@ -155,63 +159,69 @@ const HomeLatestPosts = ({classes}: {classes: ClassesType}) => {
   return (
     <AnalyticsContext pageSectionContext="latestPosts">
       <SingleColumnSection>
-        <SectionTitle title={latestPostsName} noTopMargin={isFriendlyUI} noBottomPadding>
-          <LWTooltip
-            title={`Use these buttons to increase or decrease the visibility of posts based on ${taggingNameSetting.get()}. Use the "+" button at the end to add additional ${taggingNamePluralSetting.get()} to boost or reduce them.`}
-            hideOnTouchScreens
-          >
-            <SettingsButton
-              className={classes.hideOnMobile}
-              label={filterSettingsVisibleDesktop ?
-                filterSettingsToggleLabels.desktopVisible :
-                filterSettingsToggleLabels.desktopHidden}
-              showIcon={false}
-              onClick={changeShowTagFilterSettingsDesktop}
-              textShadow={isLWorAF}
-            />
-            <SettingsButton
-              className={classes.hideOnDesktop}
-              label={filterSettingsVisibleMobile ?
-                filterSettingsToggleLabels.mobileVisible :
-                filterSettingsToggleLabels.mobileHidden}
-              showIcon={false}
-              onClick={() => {
-                setFilterSettingsVisibleMobile(!filterSettingsVisibleMobile)
-                captureEvent("filterSettingsClicked", {
-                  settingsVisible: !filterSettingsVisibleMobile,
-                  settings: filterSettings,
-                  pageSectionContext: "latestPosts"
-                })
-              }} />
-          </LWTooltip>
-        </SectionTitle>
-  
-        <AnalyticsContext pageSectionContext="tagFilterSettings">
-          <div className={classNames({
-            [classes.hideOnDesktop]: !filterSettingsVisibleDesktop,
-            [classes.hideOnMobile]: !filterSettingsVisibleMobile,
-          })}>
-            <TagFilterSettings
-              filterSettings={filterSettings} setPersonalBlogFilter={setPersonalBlogFilter} setTagFilter={setTagFilter} removeTagFilter={removeTagFilter}
-            />
-          </div>
-        </AnalyticsContext>
-        {isFriendlyUI && <StickiedPosts />}
-        <HideRepeatedPostsProvider>
-          {showCurated && <CuratedPostsList />}
-          <AnalyticsContext listContext={"latestPosts"}>
-            {/* Allow hiding posts from the front page*/}
-            <AllowHidingFrontPagePostsContext.Provider value={true}>
-              <PostsList2
-                terms={recentPostsTerms}
-                alwaysShowLoadMore
-                hideHiddenFrontPagePosts
+        <PostsListViewProvider>
```

Comment:
Most of this change is indentation - the only additions are the `PostsListViewProvider` and adding `viewType="fromContext"` to the posts list